lib/PuppeteerSharp.Tests/SingleFileDeployment/FactRunableOnWindowsAttribute.cs

```diff
@@ -0,0 +1,18 @@
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.SingleFileDeployment
+{
+    internal class FactRunableOnWindowsAttribute : FactAttribute
```

Comment:
There's other `xunit` test attributes in https://github.com/hardkoded/puppeteer-sharp/tree/master/lib/PuppeteerSharp.Tests/Attributes

Should this live in there?